reference/3.0/Microsoft.PowerShell.Core/About/about_Prompts.md

```diff
@@ -208,9 +206,9 @@ The prompt resembles the following prompt:
 03/15/2012 17:49:47>
 ```
 
-You can also change the default Prompt function:
+You can also change the default `Prompt` function:
 
-For example, the following modified Prompt function adds "[ADMIN]:" to the
+For example, the following modified `Prompt` function adds "[ADMIN]:" to the
 built-in PowerShell prompt when PowerShell is opened by using the "Run as
 administrator" option:
```

Comment:
replace quotes with bold for **Run as administrator**

Reply:
Fixed